.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
We should add this file to the `.gitignore`

Reply:
Done!